### ISSUE TITLE
La Carte n'apparait pas dans les cartes personnalisées

### ISSUE THREAD

**Issue by ConvarDeutschland** [hacf-fr/lovelace-meteofrance-weather-card#118]
Bonjour,

après installation de la carte elle apparaît bien dans les entités de HACS, mais lorsque je veux ajouter une carte personnalisée, la carte n'est pas disponible. Je peucx choisir la carte classique météo france mais pas la personnalisée

**Comment by SebastienMrc** [comment]
Bonjour, j'ai exactement le même problème... Avez-vous trouvé une solution ?

**Comment by ConvarDeutschland** [comment]
Non, désolé j'ai pas trouvé de solution

**Comment by dx44** [comment]
Bonjour,
La carte apparaît bien dans les cartes personnalisées de mon côté.
Avez-vous rafraichi l'interface de votre navigateur (CTRL+f5) ou tuée votre application mobile ? Ceci pour forcer à recharger tous les composants de l'interface.

**Comment by ConvarDeutschland** [comment]
Oui, c'est fait, mais elle n'apparaît pas. Faut il faire quelque chose après l'installation à part redémarrer ?

**Comment by fractalstore** [comment]
même problème, HA installé sous docker
